AdminUi/src/AdminUi/ClientApp/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+    "ignorePatterns": ["/*", "!/src", "!/test"],
```

Comment:
Since we don't have a `test` folder and will probably never have, you can remove it from the list of ignorePatterns